connectors/PartnerCenterReferrals/apiProperties.json

```diff
@@ -0,0 +1,49 @@
+{
+  "properties": {
+    "capabilities": [
+      "actions"
+    ],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "a45b191f-fd2a-4dfe-b883-2eb19d179ca9",
```

Comment:
@didalal Everything looks good, We will go ahead and deploy your connector's update to Production regions, but please change this to a dummy client id here? For example, a string such as this one is more suitable {{CLIENT_ID}}

Reply:
done